modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java

```diff
@@ -230,25 +230,30 @@ private long _getObjectDefinitionId(
 		Map<String, String> objectFieldTypes = stream.collect(
 			Collectors.toMap(ObjectField::getName, ObjectField::getType));
 
-		DDMFormValues ddmFormValues =
-			ddmStorageAdapterSaveRequest.getDDMFormValues();
+		for (DDMFormFieldValue ddmFormFieldValue : ddmFormFieldValues) {
+			String fieldType = ddmFormFieldValue.getType();
 
-		for (DDMFormFieldValue ddmFormValue :
-				ddmFormValues.getDDMFormFieldValues()) {
-
-			String objectFieldName = _getObjectFieldName(
-				ddmFormValue.getDDMFormField());
+			if (fieldType.equals("fieldset")) {
```

Comment:
Can you replace it with `StringUtil.equals` ?

Reply:
Sure. Already did it, and replaced the string "fieldset" with the fieldset constant.